src/yaml/builder.cr

```diff
@@ -49,9 +49,10 @@ class YAML::Builder
 
   # Creates a `YAML::Builder` that will write to the given `IO`,
   # invokes the block and closes the builder.
-  def self.new(io : IO)
+  def self.new(io : IO) : Nil
```

Comment:
General question: Should this method be named `build`? It's not a constructor and doesn't return an instance of `YAML::Builder` which I'd usually expect from a `.new` method.

Reply:
Yeah, I was surprised by this name too. We could probably rename it, but I'd like to avoid doing that in this PR.